.travis.yml

```diff
@@ -16,25 +16,8 @@ matrix:
   include:
     - python: 3.6
       env: TOXENV=quality
-    - python: 2.7
-      env: TOXENV=quality
     - python: 3.6
       env: TOXENV=docs
-    - python: 2.7
```

Comment:
We need to keep the jasmine tests. Just change the python version from 2.7 to 3.5. They should not cause any problem in case of py3.5. 
just change the python version from 2.7 to 3.5 in case of those.